graphing/src/cloudevolution/views.py

```diff
@@ -52,6 +59,45 @@ def vial_num(request, experiment, vial):
 	p.yaxis.axis_label = 'Optical Density'
 	p.line(data[:,0], data[:,1], line_width=1)
 	OD_script, OD_div = components(p)
+	od_x_range = p.x_range  # Save plot size for later
+
+	"""
+	GROWTH RATE PLOT
+	"""
+
+	gr_data = np.genfromtxt(gr_dir, delimiter=',', skip_header=2)
+
+	last_grate_update = time.ctime(os.path.getmtime(gr_dir))
+
+	p = figure(plot_width=700, plot_height=400)
+	p.y_range = Range1d(0, 1)  # Customize here y-axis range
+	p.x_range = od_x_range  # Set same size as the OD plot
+	p.xaxis.axis_label = 'Hours'
+	p.yaxis.axis_label = 'Growth rate (1/h)'
+	p.line(gr_data[:, 0], gr_data[:, 1], legend="growth rate")  # Growth rate
```

Comment:
Modified and tested. Doesn't plot anything until there's three OD values. It will always ignore the first value since it will depend on the starting OD of the experiment, which is assumed to be equal to the experiment's `lower_thresh`.